adversarial-thinking.adoc

```diff
@@ -83,19 +83,19 @@ conservative
 ____
 
 He refers to these simplistic narratives as monomania, meaning that by
-focusing only on a single thing, e.g., "just HODL", you stop thinking
+focusing only on a single thing, e.g. "just HODL", you stop thinking
 about important stuff like keeping your Bitcoin secure or doing your
-best to use Bitcoin in a trustless manner.
+best to use Bitcoin in a trustless manner.[comm: this section is a bit obscure to me. I undestand everything that is said but I don't quite get the logic behind it: what does the title refer to? And if we open by saying that adversarial thinking is relevant not only to bitcoiners, why do we only talk about Bitcoin then? And what is the logical connection between the over-simplistic narratives and thinking adversarially? I imagine the answers to all these questions but I feel like they should be more explicitly outlined in the text]
```

Comment:
I see what you mean! I start by talking about the "general public", but what I actually mean is "Bitcoin users in general, not neccesarily experts and super-nerds",

I've changed that sentence above hoping to bring clarity on what group of people I mean.

> And if we open by saying that adversarial thinking is relevant not only to bitcoiners, why do we only talk about Bitcoin then?

We are talking only about Bitcoiners, but Bitcoiners of different skill-levels.

> what is the logical connection between the over-simplistic narratives and thinking adversarially?

Only thinking about HODL, will take focus away from other stuff that's arguably more important (for you and for Bitcoin as a whole)

Do you think my changes above makes it better, and do you think we need more clarification?

Reply:
I understand the confusion, it makes so much sense now that we have better defined the "general public" of the original version! I integrated your suggestions and added a few words that I believe make the concept even clearer.
Let me know what you think about it.